feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -155,100 +163,103 @@ func configInterfaceDUT(i *oc.Interface, me, peer *attrs.Attributes, subintfinde
 }
 
 // Configure Network instance on the DUT
-func configNetworkInstance(name string, peer *attrs.Attributes) *oc.NetworkInstance {
+func configNetworkInstance(name string, intf string, id string) *oc.NetworkInstance {
 	d := &oc.Root{}
 	ni := d.GetOrCreateNetworkInstance(name)
 
 	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
-	static := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
-	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop("0")
-	ipv4Nh.NextHop, _ = ipv4Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv4)
-	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop("0")
-	ipv6Nh.NextHop, _ = ipv6Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv6)
-	ipv4Nh.Recurse = ygot.Bool(true)
-	ipv6Nh.Recurse = ygot.Bool(true)
+	niIntf := ni.GetOrCreateInterface(intf)
+	niIntf.Id = ygot.String(id)
+	niIntf.Interface = ygot.String(intf)
```

Comment:
This creates an incorrect reference to the base interface since you're passing "intfname.subif index" as an "intf" parameter to the function. Please use base-interface + subinterface index references, i.e. remove the subif index from this parameter and add niIntf.Subinterface = subif_index on the next line

Reply:
you can  refer PBR RT 3.2 for creating sub-interface inside vrf.   
https://github.com/openconfig/featureprofiles/blob/8a79c5d63cfd9f3d9580eb6701c95fc02f3ce123/feature/experimental/policy/policy_vrf_selection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go#L204

Reply:
alternately we can modify function configNetworkInstance and its call as below
in function configNetworkInstance
delete line 172
in line 171, change the argument as below 
niIntf := ni.GetOrCreateInterface(intf + "." + id)

while calling the modified method:
delete line 118
and pass arguments to the modified method as:
niConf := configNetworkInstance("VRF-10", *i2.Name, "10")

Reply:
Done in new commit.